packages/cli/src/commands/new/csharp.ts

```diff
@@ -19,10 +20,13 @@ const ArgsSchema = z.object({
 });
 
 export function CSharp(_: IContext): CommandModule<{}, z.infer<typeof ArgsSchema>> {
+  const isCSharp = Settings.load().language == 'csharp';
```

Comment:
should this throw if this isn't set? or it won't get to here?

Reply:
if this isn't set then it'll default to `teams new <typescript | csharp>`. If it is then the command becomes `teams new <typescript | csharp | <your csharp app>`.